core/templates/pages/skill-editor-page/services/skill-editor-state.service.spec.ts

```diff
@@ -229,7 +217,7 @@ describe('Skill editor state service', function() {
     };
 
     var skillDict = {
-      id: 'skill_id_1',
```

Comment:
Why have you changed this and below?

Reply:
Earlier i though skill_id are number so i changed them to 1 then i saw the backend it is used as string so i change 1 to '1'. Should i change them back to 'skill_id_1' or '1' should be fine

Reply:
Can i fix them after 6th march as my exams are going on
Thanks

Reply:
You should definitely revert back the unnecessary changes.

Sure thing, all the best for your exams :).

Reply:
changed it back to skill_id_1